packages/app/src/Feed/EventPublisher.ts

```diff
@@ -76,7 +76,7 @@ export default function useEventPublisher() {
     };
     const content = msg
       .replace(/@npub[a-z0-9]+/g, replaceNpub)
-      .replace(/note1[acdefghjklmnpqrstuvwxyz023456789]{58}/g, replaceNoteId)
+      .replace(/(\s)(note1[acdefghjklmnpqrstuvwxyz023456789]{58})(\s)/g, replaceNoteId)
```

Comment:
What if i wanted to start the note with a ref?

Reply:
do you mean like this `@note1ju6usp2u7d5v6uxxka2d37p5p400djuu555yp46h6qdrnf85k5kqg49a8c`?

Reply:
Yep